docs/advanced/custom_mapping.md

```diff
@@ -1,3 +1,5 @@
+# Custom Mapping
```

Comment:
I am 100% we will forget to add a title after we add another page and the print page would be broken since there are not tests for it. Is there any way to force this?
[markdown](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md002---first-header-should-be-a-top-level-header) lint, maybe?

Reply:
I added a check (MD041) to pre-commit and added `PULL_REQUEST_TEMPLATE.md` to ignored files